### PR TITLE
Add option to hide empty tooltips in inventories.

### PR DESCRIPTION
**Pull request by msg-programs**
Hypixel's inventory menus sometimes use items with empty names for empty slots. When the game draws the tooltip for such an item, it just draws a small dark rectangle. This adds an option to disable these empty tooltips.

### CODE REVIEW

File: src/main/java/me/xmrvizzy/skyblocker/mixin/ScreenMixin.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import net.minecraft.text.Text;
+
+@Mixin(Screen.class)
+public abstract class ScreenMixin {
+
+    @Inject(at = @At("HEAD"), method = "renderTooltip(Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/item/ItemStack;II)V", cancellable = true)
+    public void skyblocker$renderTooltip(MatrixStack matrices, ItemStack itemStack, int x, int y, CallbackInfo ci) {
+        Text stackName = itemStack.getName();
+        String strName = stackName.getString();
+        if (SkyblockerConfig.get().general.hideEmptyTooltips && strName.equals(" ")) {
```
Comment (Julienraptor01) [inline src/main/java/me/xmrvizzy/skyblocker/mixin/ScreenMixin.java:21]: add a skyblock check by changing line to
```java
if (Utils.isOnSkyblock && SkyblockerConfig.get().general.hideEmptyTooltips && strName.equals(" ")) {
```

**Review by Julienraptor01 — CHANGES_REQUESTED** [review]
pretty good 👍, work as expected except the fact it doesn't check for skyblock

**Review by AzureAaron — APPROVED** [review]

**Review by Julienraptor01 — APPROVED** [review]
👌

**Comment by LifeIsAParadox** [comment]
Resolve the conflict then I can make a merge

**Comment by msg-programs** [comment]
Fixed it, should be ready to go now.

**Pull request merged**